app.rb

```diff
@@ -261,6 +261,9 @@ def form_xml_for_settings(tag, values)
         .in_state(params[:image_state])
         .in_env(environment)
         .set_region(params[:region])
+    elsif params[:provider] == 'google'
```

Comment:
I don't like this being keyed on `'google'`; this should be a more generic behavior for any images that do not have a defined region or environment.

Reply:
Yep, same applies to Oracle and this is related to #9 when we have global images and no update infrastructure.

Reply:
I have updated the code so that no longer google is no longer hard coded